dac2bids.py

```diff
@@ -293,8 +303,12 @@ def main():
     p.add_option('--sub', '-s', default=1, type="int", help="The subject number")
     p.add_option('--ses', '-e', default=1, type="int", help="The session number")
     p.add_option('--skipfmap', '-f', action="store_true", help="Skips fieldmaps.")
+    p.add_option('--taskname', '-t', default="task-unknown", help="The task name")    
 
+    global options 	# make options globally available
     options, arguments = p.parse_args()
+    if not 'task-' in options.taskname:
+        options.taskname = 'task-'+options.taskname
```

Comment:
A better solution to the `global` issue, also for consistency with the rest of the code (the handling of subjects and sessions), is to add the taskname option as an extra keyword argument for `create_yaml`.